### DISCUSSION TITLE
自定义菜单后无法使用

### DISCUSSION THREAD

**Discussion by ctrlh985** [didi/LogicFlow#1299] [Q&A] [answered]
我在初始化的时候重新覆盖了原有的菜单选项，但是在使用的时候浏览器控制台提示找不到该方法
![image](https://github.com/didi/LogicFlow/assets/68211033/0c18376e-c2b4-4eb8-ab86-8ac930d0f2f8)

**Comment by wumail** [comment, marked as answer]
看看是不是this指向的问题
```typescript
callback: (edge) => {
  this.lf.deleteEdge(edge.id);
},
```

**Reply by ctrlh985** [reply]
是this的问题，谢谢解答
